src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,194 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RegistrationServiceTests {
+  private static RegistrationService registrationService;
+
+  @BeforeAll
+  static void setUpOnce() {
+    registrationService = new RegistrationServiceImpl();
+  }
```

Comment:
Consider using `@BeforeEach` instead of `@BeforeAll` for initializing `registrationService`. `@BeforeAll` is used for static initializations that are shared across all tests and are executed once. However, `@BeforeEach` ensures that each test gets a fresh instance of the service, preventing side effects between tests.

Reply:
changed